infra/main.tf

```diff
@@ -102,3 +102,58 @@ resource "aws_cloudwatch_event_target" "cloudwatch_event_target" {
     "table" : aws_dynamodb_table.dynamodb_table.name
   })
 }
+
+resource "aws_elasticsearch_domain" "elasticsearch_domain" {
+  domain_name           = "tf-creds-es-domain"
+  elasticsearch_version = "7.10"
+
+  cluster_config {
+    instance_type = "t3.small.elasticsearch"
```

Comment:
What is this instance will be used for ?
And what is this "cluster config" ? :thinking:

Reply:
This is an instance AWS generates for hosting the Elasticsearch stuck.  There are various options. I've used the smallest one.

Reply:
AWS uses the term cluster for the Elasticsearch (and Opnesearch) stuck.  Do not confuse with k8s cluster. Totally different things while using the same term.